md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/SubProcessAPI.java

```diff
@@ -120,6 +122,8 @@ public RestWrapper delete(
             @PathVariable("id") Integer processId, Principal principal) {
         RestWrapper restWrapper = null;
         try {
+            com.wipro.ats.bdre.md.dao.jpa.Process parentProcess=processDAO.get(processId);
+            processDAO.securityCheck(parentProcess.getProcessId(),principal.getName(),"write");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "write" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)